packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -55,6 +55,8 @@ export interface BundlingOptions {
   readonly user?: string;
 }
 
+export type DockerConsistency = 'consistent' | 'delegated' | 'cached';
```

Comment:
Can you switch to an `enum` and document the options? Would also move this just before or just after `DockerVolume`.

```ts
export enum DockerMountConsistency {
...
}
```

Reply:
Changed to enum and documented possible options. I've also introduced a DEFAULT option inside the enum just to give us flexibility to change the default consistency from "delegated" to some other value if needed in the future, without breaking existing projects.